go.mod

```diff
@@ -211,4 +211,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.2.6
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.2.8
```

Comment:
@CecileRobertMichon Do you know why this is here? It seems like a no-op and bit me when I was trying to update this.

Reply:
yes here's an explanation: https://github.com/kubernetes-sigs/cluster-api/issues/6081